src/blockchain_db/lmdb/db_lmdb.h

```diff
@@ -413,7 +420,7 @@ class BlockchainLMDB : public BlockchainDB
 
   bool prune_worker(int mode, uint32_t pruning_seed);
 
-  virtual bool is_read_only() const;
+  bool is_read_only() const;
```

Comment:
I'm not 100% sure if this change was required, but since the method is private anyway, I don't see a reason for it to be virtual anyway.